src/scales/scale.time.js

```diff
@@ -61,253 +69,288 @@ module.exports = function(Chart) {
 				month: 'MMM YYYY', // Sept 2015
 				quarter: '[Q]Q - YYYY', // Q3
 				year: 'YYYY' // 2015
-			}
+			},
```

Comment:
Of course, I mean you modified both lines, but only added comma on the second one. Anyway, that's not important!